src/main/java/decodes/sql/SqlDatabaseIO.java

```diff
@@ -2117,7 +2113,8 @@ public Connection getConnection()
 				Logger.instance().warning(
 					"SqlDatabaseIO.getConnection() Cannot get pooled connection: " + ex);
 			}
-		return _conn;
+		}
+		return new WrappedConnection(_conn, (c)->{/* do nothing */});
```

Comment:
This was required due to the changes in `PresentationGroupListIO` and the future changes.